Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -89,6 +89,11 @@ public class ServiceAccountCredential : ServiceCredential, IOidcTokenProvider, I
             /// </summary>
             public string KeyId { get; set; }
 
+            /// <summary>
+            /// Gets or sets the flag enforcing use of Self-Signed Jwt when OAuth scopes are set
```

Comment:
I'm not sure I'd use "enforcing" here:

```suggestion
            /// Gets or sets the flag preferring use of self-signed JWTs over OAuth tokens when OAuth scopes are explicitly set.
```

Reply:
thanks :)